### PR TITLE
Language specific women's groups

### PR DESCRIPTION
**Pull request by Johanna-hub**
Django Girls, Pyladies, ClojureBridge, Node Girls, Rails Girls, R-Ladies, Women who Go

Some groups had code of conduct, so I've put the link.  As these groups are worldwide have put UK or London specific chapter info where possible.

### CODE REVIEW

File: data/communities.json [hunk 1]
```diff
@@ -136,5 +133,110 @@
...
+  "description": "We are an international mentorship group with a focus on helping more women become active participants and leaders in the Python open-source community.",
+  "url": "http://www.pyladies.com",
+  "logo": "http://www.pyladies.com/assets/images/pyladies_logo.png",
+  "social": [
+    {
+      "name": "twitter",
+      "url": "https://twitter.com/PyLadiesLondon"
+    }
+  ],
+  "codeOfConduct": "http://www.pyladies.com/CodeOfConduct/",
```
Comment (eddiejaoude) [inline data/communities.json:163]: I had planned for `codeOfConduct` to be a `boolean` but I prefer your idea of putting a link 👍 . I will update the ones that don't have a CodeOfConduct to an empty string.

File: data/communities.json [hunk 2]
```diff
@@ -136,5 +133,110 @@
...
+  "social": [
+    {
+      "name": "twitter",
+      "url": "https://twitter.com/womenwhogolndn/"
+    }
+  ],
+  "codeOfConduct": false,
+  "levels": ["beginner", "intermediate", "advanced"],
+  "tags": ["Code", "Women", "Inclusion", "Diversity"]
+  },
```
Comment (eddiejaoude) [inline data/communities.json]: The final object in the collection (array) should not end with a `,`. Line `241` should be changed from `},` to just the closing bracket `}`.

**Comment by eddiejaoude** [comment]
Looks great 👍 . 

Just 1 comment on the `,` after the last object in the collection.

**Review by eddiejaoude — APPROVED** [review]
Great 👍

File: data/communities.json (hunk 1 above)
Comment (Johanna-hub) [reply data/communities.json:163]: Thanks, I should have realised that!

File: data/communities.json (hunk 2 above)
Comment (Johanna-hub) [reply data/communities.json]: Thanks!

**Pull request merged**